identity-iota/src/chain/mod.rs

```diff
@@ -1,10 +1,10 @@
 // Copyright 2020-2021 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-mod diff;
-mod document;
+mod diff_chain;
+mod document_chain;
 mod integration;
```

Comment:
This one could be renamed to `integration_chain`

Reply:
Done!